sbedecoder/schema.py

```diff
@@ -253,7 +257,10 @@ def _build_message_field(self, field_definition, offset, header_size=10, endian=
 
                 null_value = None
                 if 'null_value' in child:
-                    null_value = long(child['null_value'])
+                    if six.PY3:
```

Comment:
Actually in looking at this more closely, I don't think this even needs to be cast to a long -- we should just cast to an int.  In python 2, it will automatically get converted to a long if necessary.  Here is an example of where we would encounter this:

```xml
<type name="mantissa" description="mantissa" presence="optional" nullValue="9223372036854775807" primitiveType="int64"/>
```
`type(int('9223372036854775807')) == int` which is fine

Even for the case:

```xml
<type name="uInt64NULL" description="Unsigned Int 64 with null value" presence="optional" nullValue="18446744073709551615" primitiveType="uint64" sinceVersion="7"/>
```
where `type(int('18446744073709551615')) == long` we are fine because all we would do is compare it to another null value which would be the same type.